tests/examples/simulation/rutherford/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+################################################################################
+#Copyright(C) 2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH #
+# #
+#This software is distributed under the terms of the #
+#GNU Lesser General Public Licence(LGPL) version 3, #
+#copied verbatim in the file "LICENSE" #
+    ################################################################################
+
+#Test suite for the `FairRoot::ExRutherford` library
+
+    fairroot_add_catch2_test_suite(ExRutherford SOURCES test_FairRutherfordGeo.cxx
```

Comment:
Something is still wrong with the whitespace here.

Reply:
Fixed. I think by accident I format the file using clang-format.